src/main/java/user/createCorporatePage/exceptions/PageCreationFailed.java

```diff
@@ -0,0 +1,5 @@
+package user.createCorporatePage.exceptions;
+
+public class PageCreationFailed extends RuntimeException{
```

Comment:
I would still rename this to PageCreationFailedException to keep to convention and just make things easier to understand at first glance.

Reply:
name changed